fvtest/porttest/si.cpp

```diff
@@ -157,6 +157,14 @@ TEST(PortSysinfoTest, sysinfo_test0)
 	if (0 == strncmp(argv0, "./", 2)) {
 		argv0 = &argv0[2];
 	}
+	/* When this test is invoked from a Windows machine, the input in `argv` may have forward or back slashes. To avoid false
+     * negatives we normalize the path names to use forward slashes always.
+     */
```

Comment:
Indentation is using 5 spaces, please use 1 tab and 1 space.
```suggestion
	 * negatives we normalize the path names to use forward slashes always.
	 */
```